packages/salesforcedx-vscode-soql/package.json

```diff
@@ -71,7 +71,7 @@
     "papaparse": "^5.3.0",
     "sinon": "13.0.1",
     "typescript": "^4.7.4",
-    "vscode-extension-tester": "^4.0.2",
+    "vscode-extension-tester": "^5.0.0",
```

Comment:
I thought this might help...it didn't, but doesn't hurt so I let it to pick up the latest.